lib/screen/home_screen.dart

```diff
@@ -0,0 +1,150 @@
+import 'package:flutter/material.dart';
+import 'package:google_fonts/google_fonts.dart';
+import 'package:sochem/utils/constants.dart';
+import 'package:sochem/widgets/carousel.dart';
+import 'package:sochem/widgets/gridcards.dart';
+
+class HomeScreen extends StatefulWidget {
+  @override
+  _HomeScreenState createState() => _HomeScreenState();
+}
+
+class _HomeScreenState extends State<HomeScreen> {
+  @override
+  Widget build(BuildContext context) {
+    return MaterialApp(
+      debugShowCheckedModeBanner: false,
+      home: Scaffold(
+        backgroundColor: kBackgroundColor,
+        body: Column(
+          children: <Widget>[
+            Container(
+              decoration: BoxDecoration(
+                borderRadius: BorderRadius.only(
+                  bottomLeft: Radius.circular(30.0),
+                  bottomRight: Radius.circular(30.0),
+                ),
+                color: kPrimaryColor,
+                boxShadow: [
+                  BoxShadow(
+                    color: Colors.black,
+                    offset: const Offset(
+                      1.0,
+                      1.0,
+                    ), //Offset
+                    blurRadius: 30.0,
+                    spreadRadius: 1.0,
+                  ),
+                ],
+              ),
+              padding: EdgeInsets.only(top: 25.0),
+              height: MediaQuery.of(context).size.height * 0.30,
+              child: Column(
+                children: [
```

Comment:
Instead of using MediaQuery everytime just make a constant for scren height above and use it in the build function. Same applies to some other MediaQuery usages below.

Reply:
Done